server.js

```diff
@@ -20,6 +23,35 @@ app.get('/ping', (req, res) => {
   res.send('This is the About page');
 })
 
+
+// Connection URI
+const uri = 'mongodb+srv://NAYANKUMARRAJ:nkr2580@my-first-cluster.hz1puza.mongodb.net/?retryWrites=true&w=majority';
+
+// Create a new MongoClient
+const client = new MongoClient(uri, { useNewUrlParser: true, useUnifiedTopology: true });
+
+// Connect to the MongoDB cluster
+client.connect()
+  .then(() => {
+    console.log('Connected to MongoDB Atlas');
+    // Do something with the connected client, such as performing database operations
+    app.get('/',(req,res)=>{
+      res.json({
+        message: 'mission successful',
+        database: 'connected'
+  })
+    })
+    const database = client.db('test');
+    const collection = database.collection('users');
+    app.get('/users', async (req,res)=>{
+    const result = await collection.find({}).toArray();

```

Comment:
Add error handling for database operations. This will help the application recover gracefully from any issues. [important]